package.json

```diff
@@ -10,12 +10,14 @@
     "dev": "webpack",
     "watch": "webpack --watch",
     "test": "jest -c jest.config.js",
+    "test:snapshot": "jest -c jest-snapshot.config.js",
+    "snapshot": "start-server-and-test storybook http-get://localhost:6006 test:snapshot",
```

Comment:
What's the difference between `test:snapshot` and `snapshot`?

Reply:
I don't know why, but when i use this script:
```
 "snapshot": "start-server-and-test storybook http-get://localhost:6006 jest -c jest-snapshot.config.js",
```

snapshot command fire snapshots and all of unit tests